### ISSUE TITLE
More options for force-element-nesting

### ISSUE THREAD

**Issue by thasmo** [sasstools/sass-lint#827]
In my case I want to have elements nested **except** for two cases:

**pseuso classes**
```scss
.element:last-child
```

**chained classes**
```scss
.element.is-active
```

I'd love to see options for `force-element-nesting` to have it configured for these cases.

**Comment by ncoden** [comment]
@DanPurdy I find it useful too. I think the nesting forcing rules are too restrictives. When I have to apply properties to only one particular case of an element (pseudo-class / pseudo-element / chained class...), I do not expect to have to nest the pseudo-class/pseudo-element/class inside its element.

**Should throw an error:** factorizable multiple declaration blocks
```scss
.element:pseudo-class { ... }
.element:other-pseudo-class { ... }
```
```scss
.element.chained-class { ... }
.element:pseudo-class { ... }
.element::pseudo-element { ... }
```

**Should not throw an error:** single declaration block
```scss
.element:pseudo-class { ... }
```
```scss
.element.chained-class { ... }
.other-element.chained-class { ... }
```
